frontend/packages/console-dynamic-plugin-sdk/src/utils/k8s/hooks/useK8sModel.ts

```diff
@@ -0,0 +1,30 @@
+// eslint-disable-next-line @typescript-eslint/ban-ts-ignore
+// @ts-ignore: FIXME missing exports due to out-of-sync @types/react-redux version
+import { useSelector } from 'react-redux';
```

Comment:
can you try using newer react-redux types for `console-dynamic-plugin-sdk` only ? So we can get rid of that `ts-ignore`

Reply:
ah, yeah I tried sometime back. Let me try this agian.

Reply:
Sadly I believe it's not so easy.. My understanding is there is a heavy typescript change needed if we did that. That's why we have not updated it. There are several instances of this. It was recently brought up in [#forum-ui](https://coreos.slack.com/archives/C6A3NV5J9/p1637011656277000?thread_ts=1637006830.272900&cid=C6A3NV5J9)

Reply:
@rawagner / @andrewballantyne I tried upgrading but it's not a simple version upgrade ... typescript check started complaining across the app. If you don't have very strong concerns with having comments for now, we can let this PR go and handle it with a separate ticket/PR.

Reply:
Please leave it to another PR. @rawagner this is a long standing issue with OpenShift Console and this is not the PR to try and fix it. I [posed the question](https://coreos.slack.com/archives/C6A3NV5J9/p1637681842376100) to the Console team. I agree this needs to be fixed, but this may need to be fixed in Dynamic Plugin SDK repo and not here in OpenShift Console.